backend/PyMatcha/utils/populate_database.py

```diff
@@ -44,7 +45,11 @@ def populate_users(amount=150, drop_user_table=False):
         bio = lorem.paragraph()
 
         coords = user.get_coordinates()
-        geohash = Geohash.encode(float(coords["latitude"]), float(coords["longitude"]))
+        geohash = str(Geohash.encode(float(coords["latitude"]), float(coords["longitude"])))
+        # u0, gb, ez, sp corresponds to the start of geohash inside of France
```

Comment:
I don't know python, but maybe ("u0", "gb", "ez", "sp") could be a constant named GEOHASH_FRANCE. It then is clear what is it and it's easy to then switch to another country or set of countries. No need for comment then. Or maybe each of the (a, b, c, d) could be a constant.